### PR TITLE
Move Message Framer implementation details to implementation

### PR DESCRIPTION
**Pull request by tfpauly**
Completes #344

### CODE REVIEW

File: draft-ietf-taps-impl.md [hunk 1]
```diff
@@ -540,6 +540,143 @@ It is also possible that protocol stacks within a particular leaf node use 0-RTT
...
+~~~
+
+To provide an example, a simple protocol that adds a length as a header would receive
+the `NewSentMessage` event, create a data representation of the length of the Message
+data, and then send a block of data that is the concatenation of the length header and the original
+Message data.
+
+## Receiver-side Message Framing {#receive-framing}
+
+In order to parse an received flow of data into Messages, the Message Framer
```
Comment (abrunstrom) [inline draft-ietf-taps-impl.md]: an received --> a received

File: draft-ietf-taps-interface.md [hunk 2]
```diff
@@ -1727,7 +1728,7 @@ By default, Receive will try to deliver complete Messages in a single event ({{r
 The application can set a minIncompleteLength value to indicates the smallest partial
```
Comment (abrunstrom) [inline draft-ietf-taps-interface.md]: to indicates --> to indicate

File: draft-ietf-taps-impl.md [hunk 3]
```diff
@@ -540,6 +540,143 @@ It is also possible that protocol stacks within a particular leaf node use 0-RTT
...
+
+## Defining Message Framers
+
+A Message Framer is primarily defined by the set of code that handles events
+for a framer implementation, specifically how it handles inbound and outbound data
+parsing. One way to represent a Message Framer is as an object that an application
+or other piece of software can interact with to receive events for parsing and sending
+data. Such Message Framer objects will deliver events to code that is written either as
+part of the application or a helper library. This piece of code that implements custom
+framing logic will be referred to as the "framer implementation".
```
Comment (abrunstrom) [inline draft-ietf-taps-impl.md]: How does "Message Framer objects will deliver events to code that is written either as part of the application or a helper library" relate to the Transport Services implementation providing common framers as mentioned earlier?

Comment (tfpauly) [reply draft-ietf-taps-impl.md]: Good call. Reworded this section!

File: draft-ietf-taps-interface.md (hunk 2 above)
Comment (tfpauly) [reply draft-ietf-taps-interface.md]: Fixed

File: draft-ietf-taps-impl.md (hunk 1 above)
Comment (tfpauly) [reply draft-ietf-taps-impl.md]: Fixed

**Pull request merged**